GUI/QtQuick/qml/WalletProperties/WalletPropertiesPopup.qml

```diff
@@ -57,11 +49,20 @@ CustomPopup {
         wallet_properties_vm: root.wallet_properties_vm
 
         onSig_success: (nameExport, pathExport) => {
-            root.close_click()
+            if (export_wo_wallet.isExitWhenSuccess) {
+                root.close_click()
 
-            show_popup(qsTr("Your watching-only wallet has successfully been exported\n\nFilename:\t%1\nFolder:\t%2")
-                .arg(nameExport)
-                .arg(pathExport))
+                show_popup(qsTr("Your watching-only wallet has successfully been exported\n\nFilename:\t%1\nFolder:\t%2")
+                    .arg(nameExport)
+                    .arg(pathExport))
+            }
+            else {
+                _stack_view.pop()
+
+                show_popup(qsTr("Your watching-only wallet has successfully been exported\n\nFilename:\t%1\nFolder:\t%2")
+                    .arg(nameExport)
+                    .arg(pathExport))
```

Comment:
This looks like a duplication of the call above. Why not call it only once, regardless of condition?